spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestPositionDeletesTable.java

```diff
@@ -1278,10 +1321,10 @@ public void testNormalWritesNotAllowed() throws IOException {
 
     Dataset<Row> scanDF = spark.read().format("iceberg").load(posDeletesTableName);
 
-    Assert.assertThrows(
-        "position_deletes table can only be written by RewriteDeleteFiles",
-        IllegalArgumentException.class,
-        () -> scanDF.writeTo(posDeletesTableName).append());
+    assertThatThrownBy(
+            () -> scanDF.writeTo(posDeletesTableName).append(),
+            "position_deletes table can only be written by RewriteDeleteFiles")
```

Comment:
I don't think we need this description. It should be self-explanatory that this isn't supported once we add a exception message check